src/main/controllers/AttachmentController.ts

```diff
@@ -0,0 +1,64 @@
+import { Response } from 'express';
+
+import { AppRequest } from '../definitions/appRequest';
+import { PageUrls } from '../definitions/constants';
+import { getDocId } from '../helper/ApiFormatter';
+import { getLogger } from '../logger';
+import { getCaseApi } from '../services/CaseService';
+
+import { getClaimantResponseDocDownload } from './helpers/TseRespondentApplicationHelpers';
+
+const logger = getLogger('AttachmentController');
+
+/**
+ * Displays a supporting material document
+ */
+export default class AttachmentController {
+  public async get(req: AppRequest, res: Response): Promise<void> {
+    const docId = req.params.docId;
+    const userCase = req.session?.userCase;
+    let respondentAppDocId;
+    let claimantResponseDocId;
+    let decisionDocId;
+    let requestDocId;
+    let contactTribunalDocId;
+
+    if (req.session.documentDownloadPage === PageUrls.RESPONDENT_APPLICATION_DETAILS) {
```

Comment:
Not sure if this is the best way of doing things, but I don't want to block this any longer